lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/SharedState.hs

```diff
@@ -310,3 +347,10 @@ isShared addr st = case st of
                 (Nothing, st)
     PendingSharedState {} ->
         (Nothing, st)
+
+
+instance IsOurs (SharedState n k) Address where
+    isOurs _addr state = (Nothing, state)
+
+instance IsOurs (SharedState n k) RewardAccount where
+    isOurs _account state = (Nothing, state)
```

Comment:
Both instance seems incorrect here. How do we intend to discover addresses from a `SharedState`?

Reply:
I intend to change it in the next PR dealing with shared address discovery. And to test it then properly